### PR TITLE
config: update size plugin threshold

### PR DESCRIPTION
**Pull request by raelga**
#### What type of PR is this?

/kind feature
/priority important-longterm
/assign

#### What this PR does / why we need it:

To better fit the size of the PRs that we usually do.

### CODE REVIEW

File: manifests/prow/configmaps/plugins.yaml [hunk 1]
```diff
@@ -187,10 +187,10 @@ data:
 
     size:
         s: 8
-        m: 32
-        l: 128
+        m: 64
+        l: 512
         xl: 512
```
Comment (slopezz) [inline manifests/prow/configmaps/plugins.yaml:192]: What about XL change? now shares same number with L

Comment (raelga) [reply manifests/prow/configmaps/plugins.yaml]: fixed

**Pull request merged**